genai-on-vertex-ai/vertex-ai-search/Ingesting_unstructured_docuemtns_with_metadata.ipynb

```diff
@@ -0,0 +1,1054 @@
+{
```

Comment:
<p>Most headings are at the same level. Can you please organize them?</p><p> </p><p>e.g.</p><p> </p><ol><li>Setup environment</li><li class="ql-indent-1">Import</li><li class="ql-indent-1">Auth</li><li class="ql-indent-1">Configure Environment</li><li>Create Vertex AI Search Datastore &amp; App </li><li class="ql-indent-1">Prerequisite: GCS bucket..</li><li class="ql-indent-1">...</li><li>Prepare documents with metadata for ingestion</li><li>Ingest documents to datastore</li><li>Run queries with metadata filters</li><li>Cleanup</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>